### PR TITLE
Prevent error when accepting invite with same email that sent it

### PR DESCRIPTION
**Pull request by ambuznego**
image

### CODE REVIEW

File: assets/js/MagicRouter.jsx [hunk 1]
```diff
@@ -88,89 +92,69 @@ const MagicRouter = (props) => {
         <Route
           path="/join_organization"
           loaded={loadedOrganization}
-          component={JoinOrganizationPrompt}
+          component={(props) => (
+            <JoinOrganizationPrompt user={user} {...props} />
+          )}
```
Comment (ambuznego) [inline assets/js/MagicRouter.jsx:97]: main change here, everything else is linting

File: assets/js/components/auth/JoinOrganizationPrompt.jsx [hunk 2]
```diff
@@ -60,37 +61,49 @@ class JoinOrganizationPrompt extends Component {
...
-              ) : (
-                <Title>Searching for invite...</Title>
-              )}
+              {invite &&
+                (this.props.user.sub === invite.inviter_id ? (
+                  <div>
+                    This invite was created by the same email you're currently
+                    logged in with (<b>{this.props.user.email}</b>). You cannot
+                    accept or reject the invite under this login.
+                  </div>
```
Comment (ambuznego) [inline assets/js/components/auth/JoinOrganizationPrompt.jsx:70]: one of 2 main changes

File: assets/js/components/auth/JoinOrganizationPrompt.jsx [hunk 3]
```diff
@@ -60,37 +61,49 @@ class JoinOrganizationPrompt extends Component {
...
+                    disabled={!invite}
+                    type="primary"
+                    onClick={this.acceptInvitation}
+                    style={{ width: "100%" }}
+                  >
+                    Accept Invitation
+                  </Button>
+                </Col>
+              </Row>
+            )}
```
Comment (ambuznego) [inline assets/js/components/auth/JoinOrganizationPrompt.jsx:106]: 2 of 2 main changes

**Review by vicmgs — APPROVED** [review]
perfecto

**Pull request merged**